usagerep/usagerep.go

```diff
@@ -97,6 +97,9 @@ func (u *UsageReporter) PeriodicallyReportUsage(ctx context.Context) {
 
 	doReport := func() {
 		alpEnabled := (config["PolicySyncPathPrefix"] != "")
+		if config["BPFEnabled"] == "true" {
+			config["ClusterType"] = config["ClusterType"] + ",bpf"
```

Comment:
I don't think you should change the incoming map here since this code doesn't own it (and it may be going to multiple places).  Please can you define a new variable to modify locally.